checker/src/body_visitor.rs

```diff
@@ -161,8 +163,8 @@ impl<'analysis, 'compilation, 'tcx, E> BodyVisitor<'analysis, 'compilation, 'tcx
         function_constant_args: &[(Rc<Path>, Rc<AbstractValue>)],
         parameter_types: &[Ty<'tcx>],
     ) -> Summary {
-        if cfg!(DEBUG) {
-            let mut stdout = std::io::stdout();
+        if !cfg!(DEBUG) {
```

Comment:
If you remove the prefixed negation, all tests should pass. If you keep the negation and fix the write_mir_pretty issue, all tests should pass.